### ExprTargetDB
=====

### An Integrative Approach to Predicting Human MicroRNA Targets

Gene expression is controlled by various genetic and non-genetic factors. The binding of microRNAs (miRNAs) to their mRNA targets has recently been demonstrated to be an important mechanism of gene regulation. Individual miRNAs post-transcriptionally regulate the expression of multiple gene targets by binding to mRNA transcripts. Therefore, a comprehensive and reliable catalogue of miRNA targets is critical to understanding gene regulation. Most current miRNA target prediction algorithms take advantage of different biochemical/thermodynamic properties of the sequences of miRNAs and their gene targets. Though they have been of value to researchers, the prediction results of these methods are somewhat uncorrelated and their degree of overlap is poor. We implement here a novel approach, ExprTarget, to integrate some of the most frequently used miRNA target prediction methods as well as the expression datasets on HapMap cell lines generated in our laboratory. We conducted an analysis of its performance using the database of experimentally supported miRNA targets as gold standard and using cross-validation. Our approach greatly improves miRNA target prediction relative to the existing prediction algorithms. We also developed an online database, ExprTargetDB, of human miRNA targets predicted by this approach as well as a potential reference resource on miRNA-mediated gene regulation in the HapMap LCLs to facilitate future studies of the mechanisms of gene regulation and gene regulatory networks.

Citation Info
Gamazon ER, Im H-K, Duan S, et al. (2010) ExprTarget: An Integrative Approach to Predicting Human MicroRNA Targets.
PLoS ONE 5(10): e13534.
